src/main/java/de/tum/in/www1/artemis/domain/enumeration/ExerciseType.java

```diff
@@ -11,4 +20,19 @@ public enum ExerciseType {
     public String getExerciseTypeAsReadableString() {
         return this.toString().toLowerCase().replace('_', ' ');
     }
+
+    public String getDiscriminator() {
```

Comment:
This should get a Javadoc that explains what the discriminator is/what it is needed for. I’d not expect that to be common knowledge.